RELEASE_NOTES.md

```diff
@@ -25,6 +25,10 @@
 
 - Classes `Bounds` and `SystemBounds` now implement the `__contains__` method, allowing the use of the `in` operator to check whether a value falls within the bounds or not.
 
+## Enhancements
+
+- The resampler now shows a error message where it is easier to identify the compnent and metric when it can't find relevant data for the current resampling window.
```

Comment:
```suggestion
- The resampler now shows an error message where it is easier to identify the component and metric when it can't find relevant data for the current resampling window.
```